pkg/netpol/eval/internal/k8s/ipBlock.go

```diff
@@ -6,6 +6,7 @@ import (
 	"net"
```

Comment:
should the package for IPBlock be k8s? Seems unrelated.

Reply:
opened a separate issue #48